apps/prairielearn/src/lib/manualGrading.sql

```diff
@@ -1,4 +1,25 @@
--- BLOCK select_next_ungraded_instance_question
+-- BLOCK select_next_ai_submission_group_id
+SELECT
```

Comment:
Just want to confirm how this interacts with the `next` system:

- 2 submissions in ai_submission_group_id 1
- 2 submissions in ai_submission_group_id 2

If I click next, starting with a submission in ai_submission_group_id 2, can I ever reach the submissions in ai_submission_group_id 1?

Reply:
In my opinion, we should order the way you navigate through the ai submission groups by using the highest instance_question_id in that group.

Reply:
Also see https://github.com/PrairieLearn/PrairieLearn/issues/12005#issuecomment-3255363089 for recent discussion related to previous/next buttons.